VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -818,6 +818,17 @@ def command_output_from_subprocess(self , args, process_wait_time):
         out = process_out.stdout.read().decode()
         out = str(out)
         return out
+
+    @staticmethod
+    def split(txt):
+        result = None
+        try:
+            result = shlex.split(txt)
+        except Exception as e:
+            Utils.HandlerUtil.HandlerUtility.log("Shlex.split threw exception:" + str(e))
```

Comment:
Is this working/tested ? log does not seem to be a static method.

Reply:
+1

Reply:
updated the code and tested it. logging is working now